src/touch.js

```diff
@@ -0,0 +1,89 @@
+var common = require('./common');
+var fs = require('fs');
+
+module.exports = _touch;
```

Comment:
This should be immediately after the closing curly brace of `_touch()` if possible